schema/string_test.go

```diff
@@ -4,30 +4,31 @@ import "testing"
 
 // To be in par with the python library.
```

Comment:
Could you please add a test where the minLength fails? 

You could look at [time_test.go](https://github.com/frictionlessdata/tableschema-go/blob/master/schema/time_test.go#L19) to get ideas of what cases can be checked.

Also, could you please leave change and file in the same commit in future reviews? That would make review process even better.

Thanks a lot!

Reply:
Thanks your feedback, I have modify what you have mentioned above.
In string_test.go, I create to subtask now, one for validMinLegnth and one for Error.